libs/api/domains/communications/src/lib/communications.service.spec.ts

```diff
@@ -51,7 +55,7 @@ describe('communicationsService', () => {
     const moduleRef = await Test.createTestingModule({
       imports: [
         LoggingModule,
-        EmailModule.register({ useTestAccount: true }),
```

Comment:
What happens now when tests are ran?

Reply:
The config defaults to the values set in the email config file: `libs/email-service/src/lib/email.config.ts`

`useTestAccount` defaults to `true` but you can override it either by setting the environment variable `EMAIL_USE_TEST_ACCOUNT` to `false` or by setting the environment variable `USE_NODEMAILER_APP` to `true`.

So, in the build pipeline the test will run just as it did before. Locally, it will also run as before unless you configure it otherwise.